modules/renderer/map.js

```diff
@@ -566,6 +623,10 @@ export function rendererMap(context) {
         surface
             .classed('low-zoom', zoom <= lowzoom(lat));
 
+        var taskExtent = context.rapidContext().getTaskExtent(); 
```

Comment:
remove this line, it's now redundant with the extra defensive check at the top of isTaskRectangular().